Description: system-install: symlink to version.cfg in server install
 * This fixes broken server installs, backport material.
 Author: Martin Erik Werner "arand" <martinerikwerner@gmail.com>
 Origin: upstream, http://redeclipse.net/svn/5503
 Last-Update: 2013-03-25

--- a/src/system-install.mk
+++ b/src/system-install.mk
@@ -84,6 +84,8 @@
 		-i $(gamesbindir)/$(appname)-server
 	install -m644 ../$(appgamedir)/version.cfg \
 		$(datadir)/$(appname)/version.cfg
+	ln -s $(patsubst $(DESTDIR)%,%,$(datadir))/$(appname)/version.cfg \
+		$(libexecdir)/$(appname)/version.cfg
 
 system-install-data:
 	install -d $(datadir)/$(appname)
